Sources/TuistKit/ProjectGenerator/ProjectGenerator.swift

```diff
@@ -137,6 +139,7 @@ class ProjectGenerator: ProjectGenerating {
     }
 
     private func postGenerationActions(for graph: Graph, workspaceName: String) throws {
+        try signingInteractor.install(graph: graph)
```

Comment:
I might have missed something but, shouldn't the `install` be modeled as a side effect?

Reply:
We've had this discussion in Slack and modelling it as a side effect is not really well-suitable for side effects, as they work currently at least. But if you know of a nice way of doing so, I'd gladly refactor it as a side effect.

Reply:
Let's ship it like this and we can see if we can refactor it into a side effect.